package.json

```diff
@@ -96,7 +98,9 @@
     "react-collapse": "^5.1.0",
     "react-content-loader": "^6.1.0",
     "react-dom": "18.1.0",
+    "react-freeze": "^1.0.3",
```

Comment:
Why is this change added here?

Reply:
Answer here: https://github.com/Expensify/App/pull/20179#discussion_r1226425029.

Should I uninstall them or we can keep it?

Reply:
I think those changes are reasonable but should be handled in a separate PR to keep the scope of this one more focused 👍🏼

Reply:
I agree, removed all unrelated packages :)